.readthedocs.yml

```diff
@@ -29,4 +29,4 @@ sphinx:
 # See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
 python:
   install:
-  - requirements: doc/requirements.txt
+  - pip install ".[all]"
```

Comment:
How about `pip install ".[doc]"` or `python -m pip install ".[doc]"`? No need to spend build resources installing `numpy` when sphinx should only need the doc building requirements.

Also this line could be indented two spaces I think?

Reply:
a) let's allow the assumption that numpy and pytest are installed when building the docs, in case we or anyone else wants to eventually dynamically create content (say, a gallery or jupyter directives).  

b) It was not indented before, and I am not 100% sure what rtd expects.

c) It is not that I like rtd even a tiny amount, just trying to get the d*n thing to not fail.  I am very much in favor of dropping rtd for GitHub pages.